app/src/main/res/drawable/clear_day.xml

```diff
@@ -0,0 +1,7 @@
+<vector android:height="32dp" android:viewportHeight="64.0"
+    android:viewportWidth="64.0" android:width="32dp" xmlns:android="http://schemas.android.com/apk/res/android">
+    <group>
+        <clip-path android:pathData="M0.01,0.02h63.96v63.96h-63.96z M 0,0"/>
```

Comment:
WARNING: [VectorRaster] This tag is not supported in images generated from this vector icon for API < 21; check generated icon to make sure it looks acceptable
 Vector icons require API 21, but when using Android Gradle plugin 1.4 or higher, vectors placed in the `drawable` folder are automatically moved to `drawable-*dpi-v21` and a bitmap image is generated each `drawable-*dpi` folder instead, for backwards compatibility (provided `minSdkVersion` is less than 21.).

However, there are some limitations to this vector image generation, and this lint check flags elements and attributes that are not fully supported. You should manually check whether the generated output is acceptable for those older devices.